src/app/components/timepicker/index.html

```diff
@@ -1,14 +1,162 @@
+        
+        <sky-demo-page-content sectionHeading="Behavior and states">
+        
+            <h3>Behavior</h3>
+            
+            <h4>Automatic default</h4>
+            <sky-docs-visual-example>
+              <sky-docs-visual-example-text>
+                <p>You can choose to initialize a timepicker with a default time. This will appear in the text field and will set the selected state of the fields in the picker popup as well.</p>
+                <p>If a timepicker has no initial value, it will automatically set its own default when the picker button is selected. The default is automatically set to the next hour, on the hour. For example, if the picker button is clicked at 10:37 AM, it will set the default time to 11:00 AM.</p>
+            </sky-docs-visual-example-text>
+              <sky-docs-visual-example-image>
+                    <img src="~/assets/img/guidelines/timepicker/timepicker-behavior-placeholder.png" />
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <h4>Entering time with keyboard input</h4>
+            <sky-docs-visual-example>
+              <sky-docs-visual-example-text>
+                  <p>Time values can be entered into the input field using the keyboard and will automatically resolve to the correct format.</p>
+              </sky-docs-visual-example-text>
+              <sky-docs-visual-example-image>
+                    <img src="~/assets/img/guidelines/timepicker/timepicker-behavior-placeholder.png" />
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <h3>States</h3>
+            
+            <sky-demo-page-component-states>
+              <sky-demo-page-component-state stateName="Default" imagePath="~/assets/img/guidelines/timepicker/timepicker-state-default.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Input field focused" imagePath="~/assets/img/guidelines/timepicker/timepicker-state-focused.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Picker open" imagePath="~/assets/img/guidelines/timepicker/timepicker-state-picker.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Time entered" imagePath="~/assets/img/guidelines/timepicker/timepicker-state-entered.png"></sky-demo-page-component-state>
+              <sky-demo-page-component-state stateName="Error" imagePath="~/assets/img/guidelines/timepicker/timepicker-state-error.png"></sky-demo-page-component-state>
+            </sky-demo-page-component-states>
+        
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Layout">
+        
+              <div class="sky-text-block">
+                  <p>Use 1/2 or 1/3 width columns for timepicker fields inside modals, except when using small (300px) modals.</p>
```

Comment:
should note that if using start/end inputs they should be 10px apart.

Reply:
Good catch.